optional/capi/ext/typed_data_spec.c

```diff
@@ -30,6 +54,7 @@ static const rb_data_type_t sample_typed_wrapped_struct_data_type = {
     sample_typed_wrapped_struct_free,
     sample_typed_wrapped_struct_memsize,
   },
+  &sample_typed_wrapped_struct_parent_data_type,
```

Comment:
@ko1 Removing this line makes the new spec fail:
```
TypeError: wrong argument type sample_typed_wrapped_struct (expected sample_typed_wrapped_struct_parent)
```

Is the code in `sws_typed_get_struct_parent_type` not supported?
Is it possible to express some kind of inheritance with TypedData then?